admin/class-admin-apple-themes.php

```diff
@@ -107,18 +125,55 @@ public function page_themes_render() {
 	}
 
 	/**
+	 * Theme preview page render.
+	 *
+	 * @access public
+	 */
+	public function page_theme_preview_render() {
+		if ( ! current_user_can( apply_filters( 'apple_news_settings_capability', 'manage_options' ) ) ) {
+			wp_die( __( 'You do not have permissions to access this page.', 'apple-news' ) );
+		}
+
+		$error = '';
+		// Check for a valid theme
+		if ( ! isset( $_GET['theme'] ) ) {
+			$error = __( 'No theme was specified to preview', 'apple-news' );
+		} else {
+			$theme_name = sanitize_text_field( $_GET['theme'] );
+
+			// Load the theme
+			$theme = get_option( $this->theme_key_from_name( $theme_name ) );
+			if ( empty( $theme ) || ! is_array( $theme ) ) {
+				$error = sprintf(
```

Comment:
The `$error` variable is used on the actual template page referenced on line 158.